### ISSUE TITLE
Xp gaining bug and lycanite incopatibility

### ISSUE THREAD

**Issue by Zack7292** [TheXFactor117/Loot-Slash-Conquer-Pre1.14#62]
1. when you add zeros into the config of xp you gain and go kill a monster around lvl 50 Your xp will glitch.

2. its not compatible with lycanite :/

**Comment by Rensik** [comment]
1. - How many Zeros did you put???

2. - alpha 6.3 will have the fix for Lycanite's.

**Comment by MangaBurger** [comment]
I do not know anything about the mod when you mess with the configs... but I currently have put some mods together to create a map. Among those are LSC and Lycanite. With standard configs on both the mods work fine.

**Comment by Zack7292** [comment]
mmmh idk since its was pretty mush at the time of the last realease, maybe this amount: 0000000000000000  (srry for the waiting i forgot i have psoted this xD)

**Comment by Rensik** [comment]
Yeah. Don't put that many Zeros. 2 or 3, maybe might be an issue, but 10 or more? Yeah, not gonna do anything about that. Honestly, any changes you make beyond default in the config should be an at-your-own-risk kind of situation. So yeah, make changes 'at-your-own-risk'.

**Comment by Zack7292** [comment]
yea xD i think atth its jsut bcs minecraft wasn't able to calculate the number of xp gain at all at this point
